modules/virt-creating-virtualmachineexport.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-export-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-virtualmachineexport_{context}"]
```

Comment:
is this ID OK to you? I wasn't sure if you wanted to add another dash between virtualmachine and export, or like virtual-machine-export, maybe.

Reply:
This ID is correct. The procedure is for creating a VirtualMachineExport object, not for exporting a VirtualMachine object.